src/Controller.cpp

```diff
@@ -214,8 +214,8 @@ namespace geopm
     Controller::Controller(std::shared_ptr<Comm> comm,
                            PlatformIO &plat_io,
                            const std::string &agent_name,
-                           int num_send_down,
                            int num_send_up,
+                           int num_send_down,
```

Comment:
These should map to the number of agent sample outputs (going up) and agent policy inputs (going down) but the only changes to callers of this function are in test code. Check to make sure there aren't other users of the constructor that need to be updated.

Reply:
Found one spot in the code that needed to be updated. Should be good to go now.